### PR TITLE
Update the document schema and added some routes for callback url

### PR DESCRIPTION
**Pull request by ankitaShrma**
Changed schema for document
Updated config file 
Added some routes for multi callback url support
@Anthony-Michel @nishlashakya @hung-chou

### CODE REVIEW

File: k8s/dev/api-portal-configmap.yaml [hunk 1]
```diff
@@ -4,8 +4,8 @@ metadata:
   name: api-portal-config
   namespace: dev
 data:
-  MONGO_SERVER: mongodb+srv://apiPortal:dragonlaw@cluster0-h8ub3.mongodb.net/apiportal?retryWrites=true&w=majority
+  MONGO_SERVER: mongodb://mongo-svc/apiportal
   PORT: "3000"
   ZEGAL_CLIENT: "https://zegal-client.s3.ap-south-1.amazonaws.com/dev/zegal.js"
-  CLIENT_KEY: pk_8e367994-27df-43ee-8a16-f0fb1c513ce3
+  CLIENT_KEY: pk_35f9cc27-fc94-4d8d-9bf6-2978a3a1bdee
```
Comment (Anthony-Michel) [inline k8s/dev/api-portal-configmap.yaml]: you sure you want that to be publicly visible ?

File: models/document.model.js [hunk 2]
```diff
@@ -4,7 +4,12 @@ var mongoose = require("mongoose");
 var Document = mongoose.Schema(
   {
     title: { type: String, required: true },
-    json: { type: mongoose.Schema.Types.Mixed },
+    docxPointer: { type: String },
+    pdfPointer: { type: String },
+    signers: { type: mongoose.Schema.Types.Mixed },
```
Comment (Anthony-Michel) [inline models/document.model.js]: please don't use Mixed type, define what is expected

File: k8s/dev/api-portal-configmap.yaml (hunk 1 above)
Comment (ankitaShrma) [reply k8s/dev/api-portal-configmap.yaml]: Added in secrets

File: models/document.model.js (hunk 2 above)
Comment (ankitaShrma) [reply models/document.model.js]: done

File: k8s/dev/api-portal-configmap.yaml [hunk 3]
```diff
@@ -4,8 +4,7 @@ metadata:
   name: api-portal-config
   namespace: dev
 data:
-  MONGO_SERVER: mongodb+srv://apiPortal:dragonlaw@cluster0-h8ub3.mongodb.net/apiportal?retryWrites=true&w=majority
+  MONGO_SERVER: mongodb://mongo-svc/apiportal
   PORT: "3000"
   ZEGAL_CLIENT: "https://zegal-client.s3.ap-south-1.amazonaws.com/dev/zegal.js"
-  CLIENT_KEY: pk_8e367994-27df-43ee-8a16-f0fb1c513ce3
```
Comment (Anthony-Michel) [inline k8s/dev/api-portal-configmap.yaml:10]: change this key in the system now that it's in the git history please

File: models/document.model.js [hunk 4]
```diff
@@ -4,7 +4,34 @@ var mongoose = require("mongoose");
...
-    json: { type: mongoose.Schema.Types.Mixed },
+    docxPointer: { type: String },
+    pdfPointer: { type: String },
+    signers: [
+      {
+        _id: { type: String },
+        key: { type: String }, // Signature Key to store in DocumentSignature Schema
+        party: { type: String }, // Id of party from doctype
+        email: { type: String },
+        type: { type: String }, // Signers type -> [DEFAULT_TYPE, PRIMARY_TYPE, COUNTER_TYPE, LEGACY_TYPE]
```
Comment (Anthony-Michel) [inline models/document.model.js:15]: do we need that ?

File: models/document.model.js [hunk 5]
```diff
@@ -4,7 +4,34 @@ var mongoose = require("mongoose");
 var Document = mongoose.Schema(
   {
     title: { type: String, required: true },
-    json: { type: mongoose.Schema.Types.Mixed },
+    docxPointer: { type: String },
+    pdfPointer: { type: String },
+    signers: [
+      {
+        _id: { type: String },
+        key: { type: String }, // Signature Key to store in DocumentSignature Schema
```
Comment (Anthony-Michel) [inline models/document.model.js:12]: do we need that ?

File: models/document.model.js [hunk 6]
```diff
@@ -4,7 +4,34 @@ var mongoose = require("mongoose");
...
+          surname: { type: String },
+        },
+        role: { type: String },
+        phrase: { type: String },
+        invited_at: { type: String },
+        signing_information: { type: String },
+        note: [{ message: { type: String }, _id: { type: String } }],
+        decline_note: [{ message: { type: String }, _id: { type: String } }],
+        status: { type: String },
+        signature: mongoose.Schema.Types.Mixed,
```
Comment (Anthony-Michel) [inline models/document.model.js:28]: not a mixed type anymore, please update

**Review by nishlashakya — APPROVED** [review]

**Pull request merged**